src/assets/_project/_blocks/components/buttons/_qg-buttons.scss

```diff
@@ -83,9 +83,11 @@
     @include all-states {
       border: 3px solid $qg-outline-dark;
     }
-    @include qg-underline-on-highlight-decoration {
-      color: $qg-outline-dark-hover;
+    @include on-hover-or-active {
+      border-color: $qg-outline-dark-hover !important;
```

Comment:
Isn't there any alternative to using `!important`? It's a rather blunt instrument...

Reply:
It's updated now without using !important.